java/ql/lib/ext/java.lang.model.yml

```diff
@@ -77,6 +77,8 @@ extensions:
       - ["java.lang", "Exception", False, "Exception", "(String)", "", "Argument[0]", "Argument[this].SyntheticField[java.lang.Throwable.message]", "value", "manual"]
       - ["java.lang", "Exception", False, "Exception", "(String,Throwable)", "", "Argument[0]", "Argument[this].SyntheticField[java.lang.Throwable.message]", "value", "manual"]
       - ["java.lang", "Exception", False, "Exception", "(String,Throwable)", "", "Argument[1]", "Argument[this].SyntheticField[java.lang.Throwable.cause]", "value", "manual"]
+      - ["java.lang", "Exception", False, "Exception", "(Throwable)", "", "Argument[0]", "Argument[this].SyntheticField[java.lang.Throwable.cause]", "value", "manual"]
+      - ["java.lang", "Exception", False, "Exception", "(Throwable)", "", "Argument[0].SyntheticField[java.lang.Throwable.message]", "Argument[this].SyntheticField[java.lang.Throwable.message]", "taint", "manual"]
```

Comment:
Why is this a `taint` model?

Reply:
Good question. It's [implemented](https://github.com/openjdk/jdk17u/blob/b78a848cc7ae5787d9ec9ea0ce843cd63b06efec/src/java.base/share/classes/java/lang/Exception.java#L103) as a call to the equivalent constructor of the base class `Throwable`, so I guess I was copying the model for that (also in this PR). And that is [implemented](https://github.com/openjdk/jdk17u/blob/b78a848cc7ae5787d9ec9ea0ce843cd63b06efec/src/java.base/share/classes/java/lang/Throwable.java#L317) using `toString()`, which [prepends the class name in front of the message](https://github.com/openjdk/jdk17u/blob/b78a848cc7ae5787d9ec9ea0ce843cd63b06efec/src/java.base/share/classes/java/lang/Throwable.java#L497). So I guess it's the prepending the class name which changes the string.

Reply:
Fair :)